crypto/ec/ec_asn1.c

```diff
@@ -792,18 +794,79 @@ EC_GROUP *EC_GROUP_new_from_ecparameters(const ECPARAMETERS *params)
         goto err;
     }
 
+    /*
+     * Check if the explicit parameters group just created matches one of the
+     * built-in curves.
+     *
+     * We create a copy of the group just built, so that we can remove optional
+     * fields for the lookup: we do this to avoid that one of the optional
```

Comment:
_to avoid **the possibility** that one of the optional parameters_

Reply:
Thanks! Fixing now!

Reply:
Fixed by d3da6adacc333d1ac4db955a02bca29748dd4640